runtime/mux.go

```diff
@@ -101,6 +107,16 @@ func WithOutgoingHeaderMatcher(fn HeaderMatcherFunc) ServeMuxOption {
 	}
 }
 
+//WithOutgoingTrailerMatcher returns a ServeMuxOption representing a trailerMatcher for outgoing response from gateway.
+//
+//Similar to func WithOutgoingHeaderMatcher
+//Best practice: omit "TE" header if no header or trailers need be forwarded from gRPC context.
```

Comment:
Please add a space after the slashes
```suggestion
// WithOutgoingTrailerMatcher returns a ServeMuxOption representing a trailerMatcher for the outgoing response from the gateway.
//
// This matcher will be called with each trailer in the response trailer metadata. If matcher returns true, that header will be
// passed to the http response returned from the gateway. To transform the trailer before passing to response,
// the matcher should return a modified header.
// Best practice: omit "TE" header if no trailers need to be forwarded from the gRPC server.
```